pkg/config/config.go

```diff
@@ -135,10 +134,9 @@ type Roadmap struct {
 }
 
 type Kessel struct {
-	Server   string        `mapstructure:"server"`
-	Auth     KesselAuth    `mapstructure:"auth"`
-	Insecure bool          `mapstructure:"insecure"`
```

Comment:
kessel.insecure is apparently only for local development, so I removed it in favor of us just using kessel.auth.enabled